mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -483,9 +539,16 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD i setInvisible (Z)V
 		ARG 1 invisible
 	METHOD j stopRiding ()V
+	METHOD j teleport (DDD)V
+		ARG 1 destX
+		ARG 3 destY
+		ARG 5 destZ
 	METHOD j getCameraPosVec (F)Lcry;
 		ARG 1 tickDelta
 	METHOD j allowsPermissionLevel (I)Z
+		ARG 1 permissionLevel
+	METHOD j getHardCollissionBox (Laij;)Lcrt;
+		ARG 1 colliddingEntity
```

Comment:
Typo (colliding)

Reply:
Why do I always do this? >.> I get confused about what does and does not have double-Ds and Ss.